### PR TITLE
Fix #148 - allows spaces in search terms for filtering by tag and ingredient

### LINKED ISSUE #148: Filter terms cannot contain spaces
might make it hard(er) to specifically search for ingredients that contain spaces

### PR DESCRIPTION
**Pull request by hjl99**

### CODE REVIEW

File: src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java [hunk 1]
```diff
@@ -139,8 +139,8 @@
 
             if (s.isEmpty()) {
                 return Optional.of(String.format("%s cannot be empty", kind));
-            } else if (s.split("\\s+").length > 1) {
-                return Optional.of(String.format("Filter terms should not contain spaces"));
+            } else if (kind == "Expiry date" && s.split("\\s+").length > 1) {
```
Comment (zhiayang) [inline src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java:142]: this should be `kind.equals("Expiry date")`

**Review by zhiayang — CHANGES_REQUESTED** [review]

File: src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java (hunk 1 above)
Comment (hjl99) [reply src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java:142]: Okie will change

Comment (hjl99) [reply src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java]: Okie changed, thankss

Comment (hjl99) [reply src/main/java/chopchop/logic/parser/commands/FilterCommandParser.java]: Thanks, it's updated now

**Review by zhiayang — APPROVED** [review]

**Comment by zhiayang** [comment]
👍

**Pull request merged**